options/locale/locale_en-US.ini

```
@@ -1012,6 +1013,8 @@ blame.ignore_revs = Ignoring revisions in <a href="%s">.git-blame-ignore-revs</a
 blame.ignore_revs.failed = Failed to ignore revisions in <a href="%s">.git-blame-ignore-revs</a>.
 author_search_tooltip = Shows a maximum of 30 users
 
+tree_path_not_found = Path %[1]s doesn't exist in %[2]s %[3]s
```

Accordingly, this would then be
```suggestion
tree_path_not_found_commit = Path %[1]s doesn't exist in commit %[2]s
tree_path_not_found_branch = Path %[1]s doesn't exist in branch %[2]s
tree_path_not_found_tag = Path %[1]s doesn't exist in tag %[2]s
```